README.md

```diff
@@ -57,6 +67,10 @@ Modify your project's `package.json` so that the `jest` section looks something
 ```
 This setup should allow you to write Jest tests in Typescript and be able to locate errors without any additional gymnastics.
 
```

Comment:
I don't think it's necessary. I think npm is still the standard and anyone who uses yarn knows how to translate the calls.